library/common/android_jni_utils.cc

```diff
@@ -0,0 +1,21 @@
+#include <android/log.h>
+#include <ares.h>
+#include <jni.h>
+
+// NOLINT(namespace-envoy)
+
+jint init_jvm(JavaVM* static_jvm, jobject connectivity_manager) {
```

Comment:
```suggestion
jint init_jvm(JavaVM* jvm, jobject platform_context) {
```

The `jvm` isn't static here, it's just called that in the interface layer because it's a static ref. You can add a comment that says the `context` is really a `ConnectivityManager`, but this allows it to be named something generic in the top-level interface.

Reply:
Actually I'd also call this method something else since we're not really initializing the jvm here. Maybe `platform_setup`?